server/activate_hooks.go

```diff
@@ -86,3 +94,19 @@ func (p *Plugin) OnDeactivate() error {
 
 	return nil
 }
+
+func (p *Plugin) checkRequiredServerConfiguration() (bool, error) {
+	path, err := p.API.GetBundlePath()
+	if err != nil {
+		return false, errors.Wrap(err, "failed to get bundle path")
+	}
+
+	manifest, _, err := model.FindManifest(path)
+	if err != nil {
+		return false, errors.Wrap(err, "failed to find manifest file")
+	}
+
+	req := manifest.RequiredConfig
```

Comment:
nit: `req` reads to me as `request`. Should we just pass `manifest.RequiredConfig` directly on line 111?